app/views/roles/assign.html.erb

```diff
@@ -1,6 +1,6 @@
 
 
-
+<% if current_user.try(:admin?) %>
```

Comment:
wtf is this try admin thing.

Reply:
is this how you checked for admin rights in other places also?

Reply:
did you understand the concept of how roles are being or are to be checked?

Reply:
this was already there at so many places. You can remove this. It doesn't matter because i have already added a before action.

Reply:
Yeah it was there 'BEFORE'.
Your job is to replace it with the new method of checking roles.
Now you're just using the old method more. Queer.